src/glTypes.jl

```diff
@@ -9,7 +9,7 @@ const GLdouble = Cdouble
 const GLushort = Cushort
 const GLuint = Cuint
 const GLint64 = Clonglong
-const GLbyte = Cchar
+const GLbyte = Int8
```

Comment:
Perhaps worth adding a comment to guard against regression given this doesn't have test coverage as CI isn't run on aarch64?

```suggestion
const GLbyte = Int8 # Cannot be a Cchar as Cchar is unsigned on aarch64
```

Reply:
well, I'm not really sure why we initially used Cchar here, so there is a big chance, that that was just completely wrong...
So I don't think we actually need to motivate that we don't use Cchar anymore.
See the new Clang wrapped constants, that use the OpenGL headers as the source of trueth: https://github.com/JuliaGL/ModernGL.jl/blob/4740d6ed3852238ba0f02b03864a8a582eea129f/src/gl.jl#L15
https://github.com/JuliaGL/ModernGL.jl/blob/4740d6ed3852238ba0f02b03864a8a582eea129f/src/gl.jl#L1101
Which does seem to prove, that Cchar was simply just wrongly translated early on.

Reply:
Yeah, fair